train_segmentation_model.py

```diff
@@ -77,26 +73,22 @@ def train(gcp_bucket, config_file):
         target_size=target_size,
         batch_size=batch_size,
         shuffle=True,
-        random_rotation=train_config['augmentation']['random_90-degree_rotations'],
-        seed=None)
+        random_rotation=train_config['data_augmentation']['random_90-degree_rotations'],
+        seed=train_config['training_data_shuffle_seed'])
```

Comment:
seed noted

Reply:
yeah exactly -- or maybe I'm not quite understanding your comment from above?

Reply:
all good. my lack of python know how